packages/langium/src/parser/lexer.ts

```diff
@@ -31,10 +33,12 @@ export interface Lexer {
 export class DefaultLexer implements Lexer {
 
     protected chevrotainLexer: ChevrotainLexer;
+    protected tokenBuilder: TokenBuilder;
     protected tokenTypes: TokenTypeDictionary;
 
-    constructor(services: LangiumCoreServices) {
-        const tokens = services.parser.TokenBuilder.buildTokens(services.Grammar, {
+    constructor( services: LangiumCoreServices) {
```

Comment:
Why the extra space after `(`?

Reply:
Ah, this was a pure oversight ;-) Thanks for cacthing that!